tests/plugins/test_resolve_module_compose.py

```diff
@@ -108,7 +108,11 @@ def test_resolve_module_compose(tmpdir, docker_tasker, specify_version):
     def handle_composes_post(request):
         assert request.headers['OIDC_access_token'] == 'green_eggs_and_ham'
 
-        body_json = json.loads(request.body.decode())
+        if isinstance(request.body, str):
```

Comment:
`isinstance(_, str)` is usually unsafe because of the differences between Python versions. Is there any other way of phrasing this? The example usage says it should be fine to give directly to `json.loads()`: https://github.com/getsentry/responses/blob/master/README.rst

Reply:
Maybe use `six.text_type`?

Reply:
Good idea, `six.text_type` worked